miner/payload_building_test.go

```diff
@@ -294,6 +310,31 @@ func testBuildPayload(t *testing.T, noTxPool, interrupt bool, params1559 []byte)
 	}
 }
 
+func testDAFilters(t *testing.T, maxDATxSize, maxDABlockSize *big.Int, expectedTxCount int) {
+	t.Parallel()
+	db := rawdb.NewMemoryDatabase()
+	config := holoceneConfig()
+	w, b := newTestWorker(t, config, ethash.NewFaker(), db, 0)
+	w.SetMaxDASize(maxDATxSize, maxDABlockSize)
+	const numTxs = 256
```

Comment:
if you move this const out of the function as e.g. `daFilterTestNumTxs` you can reference it in `TestDAFilters` instead of using magic 257s there.

Reply:
done